server/src/resolvers/MemberResolver.ts

```diff
@@ -0,0 +1,34 @@
+import { Args, Mutation, Ctx, Query, Resolver, Authorized } from 'type-graphql';
+
+import Member from '../entities/Member';
```

Comment:
We should setup something to use absolute path instead of relative path, it's so much easier to use :D

Reply:
Nice, i will do it in the next PR